### PR TITLE
Create a new Angular 11 app with angular CLI

### PR DESCRIPTION
**Pull request by AbdoulayeNDAO121**
Pull request of the step 1

### CODE REVIEW

File: e2e/protractor.conf.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    require('ts-node').register({
+      project: require('path').join(__dirname, './tsconfig.json')
+    });
+    jasmine.getEnv().addReporter(new SpecReporter({
+      spec: {
+        displayStacktrace: StacktraceOption.PRETTY
+      }
+    }));
+  }
+};
```
Comment (morlombakop) [inline e2e/protractor.conf.js:37]: We need a new line here, You can configure your text editor or IDE to automatically add a new line at the end of each file.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.ts]
+quote_type = single
+
+[*.md]
+max_line_length = off
```
Comment (morlombakop) [inline .editorconfig:15]: This should not be off. Set it to 120 and let's ensure in the next PR that it is the same value for the linter(s).

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
```
Comment (morlombakop) [inline .editorconfig:8]: Please ensure your editor use this config. You don't have a new line in ` e2e/protractor.conf.js`

**Review by morlombakop — COMMENTED** [review]
Good work @AbdoulayeNDAO121. Nice progress and cool speed.

Please follow the semantic-commit-message as described here: https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716

You can open your terminal, navigate to your working directory and run 
`git commit -am "feat(repository-setup):  create a new angular 11 app [295]"`

Please notice taiga ticket number in the commit message. An kindly always add it in commit messages.

Please do rename your branch to `feature/vaana-295-initialise-client-frontend-1`

It is always helpful to add a link to taiga ticket number an screenshot in the PR description.

File: e2e/protractor.conf.js (hunk 1 above)
Comment (AbdoulayeNDAO121) [reply e2e/protractor.conf.js:37]: Copy !

File: .editorconfig (hunk 3 above)
Comment (AbdoulayeNDAO121) [reply .editorconfig:8]: Ok!

**Comment by morlombakop** [comment]
@thierrymonthe  please restore this PR.

**Pull request merged**